.ansible-lint

```diff
@@ -0,0 +1,2 @@
+exclude_paths:
+  - .github/
```

Comment:
```suggestion
- .github/
```
https://github.com/ansible/ansible-lint/blob/master/.yamllint#L2

Reply:
```
indent-sequences: false
```

Aww :(

Reply:
It's more readable. B/c the indentation is for mappings.

Reply:
@webknjaz While I dislike the non-indented YAML, I can understand that is the current standard used in this repo and I will fix it. That debate is not very far from tabs vs spaces one :)

Still, I would prefer to avoid human-comments related to formatting for which there are existing linters available (yamllint).